pkg/kubelet/cadvisor/cadvisor_linux.go

```diff
@@ -31,12 +31,6 @@ import (
 	_ "github.com/google/cadvisor/container/crio/install"
 	_ "github.com/google/cadvisor/container/systemd/install"
 
-	// Register cloud info providers.
-	// TODO(#68522): Remove this in 1.20+ once the cAdvisor endpoints are removed.
-	_ "github.com/google/cadvisor/utils/cloudinfo/aws"
-	_ "github.com/google/cadvisor/utils/cloudinfo/azure"
-	_ "github.com/google/cadvisor/utils/cloudinfo/gce"
```

Comment:
https://github.com/kubernetes/kubernetes/issues/68522#issuecomment-724928827 doesn't look like it is completed yet... is it?

Reply:
oh, nm... I see this is just moving files